docs/07_how-to/36_jms.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: 'How to consume from and publish to a JMS Provider'
+sidebar_label: 'Consume and publish JMS'
+id: ht-jms
+keywords: [jms, ibm, mq, data pipeline]
+tags:
+- jms
+- ibm
+- mq
+- data pipeline
+---
+
+IBM Message Queue enables different applications to send and receive messages reliably through message queues.
+
+The Genesis Platform enables you to consume from and publish to any JMS provider, including IBM.
+
+These message queues are often used to access trade data, for example. You can create pipelines to consume trade data directly from the message queue and into your Genesis application.
+
+## Example
+
+We have created an [example application](https://github.com/genesiscommunitysuccess/howto-jms-pipelines/tree/main) that shows how to publish to an IBM message queue and consume incoming messages.
```

Comment:
Similar here, can we say "publish to a JMS queue, in this example we are using IBM" instead of leading with IBM